index.html

```diff
@@ -209,30 +261,41 @@ <h4 id="about-heading">LET'S CONNECT</h4>
-            <div>
-                <ul class="format-ul">
+            <div class="skill-list">
+                <ul>
+                    <li class="skill-item">
+                        <p>Languages</p>
+                        <img src="./assets/images/arrow-down.png" alt="arrow icon">
+                    </li>
                     <li>
-                        <div id="prog-languages" class="list-languages">
-                            <h2>Languages</h2>
-                            <img src="./assets/images/arrow-down.png" alt="down-arrow">
-                        </div>
-                        <ul id="id-list-languages">
-                            <li><img src="./assets/images/logo-js.png" alt="javascript"><span>JavaScript</span></li>
-                            <li><img src="./assets/images/logo-html.png" alt="html"><span>HTML</span></li>
-                            <li><img src="./assets/images/logo-css.png" alt="css"><span>CSS</span></li>
+                        <ul id="language-list">
+                            <li class="language-item">
+                                <div>
+                                    <img src="./assets/images/logo-js.png" alt="js icon">
+                                    <p>Javascript</p>
+                                </div>
+                            </li>
+                            <li class="language-item">
+                                <div>
+                                    <img src="./assets/images/logo-html.png" alt="html icon">
+                                    <p>HTML</p>
+                                </div>
+                            </li>
+                            <li class="language-item">
+                                <div>
+                                    <img src="./assets/images/logo-css.png" alt="css icon">
+                                    <p>CSS</p>
+                                </div>
+                            </li>
                         </ul>
                     </li>
-                    <li>
-                        <div class="list-languages separator">
-                            <h2>Frameworks</h2>
-                            <img src="./assets/images/arrow-right.png" alt="right-arrow">
-                        </div>
+                    <li class="skill-item bottom-bar">
+                        <p>Frameworks</p>
+                        <img src="./assets/images/right-arrow.png" alt="arrow icon">
                     </li>
-                    <li>
-                        <div class="list-languages separator">
-                            <h2>Skills</h2>
-                            <img src="./assets/images/arrow-right.png" alt="right-arrow">
-                        </div>
+                    <li class="skill-item bottom-bar">
+                        <p>Skills</p>
+                        <img src="./assets/images/right-arrow.png" alt="arrow icon">
                     </li>
                 </ul>
```

Comment:
- [ ] Please get in touch button should be aligned to the center as in the design 
Your code 
![image](https://user-images.githubusercontent.com/74744787/229237755-af319546-a4e5-4dc0-8cb7-8afdfdc0f35d.png)
Figma design 
![image](https://user-images.githubusercontent.com/74744787/229237791-38afc34c-b72f-42a5-9866-3a4ba94bda0c.png)

Reply:
You're right, brother @MeqdamAlqudah 

Here are how I fixed that issue ....

**HTML CODE**

In **index.html** I put this code:

                    <li **id="button-container"**>
                        <button type="submit" class="btn">
                          Get in touch
                        </button>
                    </li>

And in **styles.css** I put this another one:

  #button-container {
    display: flex;
    justify-content: center;
  }

Inside the desktop media-query to don't damage my mobile version. This is the result:

![Form-button-centered](https://user-images.githubusercontent.com/86327867/229303182-b6bc6856-9e41-4b6c-aaff-e8cfc246f51f.jpg)

Thanks for your help, dear @MeqdamAlqudah 

Javier.-

Reply:
Hi @jlcarrascof  sorry for the confusion, I'm talking about the background for the headline section which is not gray.
The background of the work section should be gray.

You can refer to my comment here:
> @jlcarrascof This problem happens because you're using the mobile background image for desktop version. On desktop version the background image should change. **We can see on this screenshot that the mobile version background image is used even on screen size > 780px** ![image](https://user-images.githubusercontent.com/74411135/229304448-739cd26b-746d-4524-b317-665d2a9f9878.png)